### ISSUE TITLE
[Accessibility] CSS: Lack of contrast between background colour and hyperlinks.

### ISSUE THREAD

**Issue by JordyToke** [JordyToke/pushedRepo#1]
The dark grey #222 background colour and default link colours do not have enough contrast to make them easily readable.
